test/modules/gs1-importer.test.js

```diff
@@ -52,7 +54,7 @@ describe('GS1 Importer tests', () => {
             injectionMode: awilix.InjectionMode.PROXY,
         });
 
-        const graphStorage = new GraphStorage(buildSelectedDatabaseParam(databaseName));
+        graphStorage = new GraphStorage(buildSelectedDatabaseParam(databaseName));
```

Comment:
Not needed to be declared as global inside test. You drop all other and graphStorage as global and only use container. For example container.graphStorage will give you desired.

Reply:
discussed